### PR TITLE
add ini support

### PR DESCRIPTION
**Pull request by boranx**
This PR implements `ini` support to Conftest requested in #45.
I put `grafana.ini` to `examples/ini` folder. Added some tests to the policy that ensures Grafana installed and configured correctly.

### CODE REVIEW

File: pkg/parser/ini/ini_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		t.Fatalf("parser should not have thrown an error: %v", err)
+	}
+
+	if input == nil {
+		t.Error("There should be information parsed but its nil")
+	}
+
+	inputMap := input.(map[string]interface{})
+	item := inputMap["Local Varaibles"]
+	fmt.Println(item.(map[string]interface{}))
```
Comment (Blokje5) [inline pkg/parser/ini/ini_test.go]: I guess this is a leftover from debugging locally. You can remove fmt entirely from the test.

File: examples/ini/grafana.ini [hunk 2]
```diff
@@ -0,0 +1,679 @@
+##################### Grafana Configuration Defaults #####################
```
Comment (Blokje5) [inline examples/ini/grafana.ini]: The example is a bit long. Readability would already improve by removing the empty variables.

File: pkg/parser/ini/ini_test.go (hunk 1 above)
Comment (boranx) [reply pkg/parser/ini/ini_test.go]: Nice catch @Blokje5, removing it.

File: examples/ini/policy/deny.rego [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package main
+
+
```
Comment (Blokje5) [inline examples/ini/policy/deny.rego]: Extra newline

**Comment by boranx** [comment]
Could you review again?

**Comment by garethr** [comment]
Looks good. Thanks.

**Pull request merged**